pkg/pillar/devicenetwork/pbr.go

```diff
@@ -88,12 +88,97 @@ func makeNetlinkRule(ifindex int, p net.IPNet, addForSubnet bool) *netlink.Rule
 	return r
 }
 
+func makeNetlinkGatewayRule(subnet net.IPNet, gateway net.IP, prio int) *netlink.Rule {
```

Comment:
Is this needed so the apps can still reach the DNS server etc at the bridge IP address?

Reply:
Yes, this is needed for the apps to reach EVE hosted service like DNS etc.